freedomofpress/securedrop-docs
# Consolidate hardware checklist into main docs

## Status
Ready for review


## Description of Changes

* Description: removed checklists section

* Fixes Issue# #220


## Testing

* check the hardware page
* check the navigation bar

## Release 

* I think it's ready to be merged in main


## Checklist (Optional)

- [X] Doc link linting (`make docs-linkcheck`) passed
